composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "jawira/case-converter": "^1.1",
-        "rector/rector-prefixed": "0.6.13"
+        "rector/rector-prefixed": "~0.7.16"
```

Comment:
Should we use `^0.7.16`? This will work up to but not including `0.8`. The `~` seems to be a little different before and after `1.0` releases.

Reply:
I recommend using this, because the sooner you discover BC break, the sooner I can help you fix it.